host/integration_test.go

```diff
@@ -1007,6 +997,13 @@ func (s *integrationSuite) TestCronWorkflow() {
 		time.Sleep(200 * time.Millisecond)
 	}
 	s.NotNil(closedExecutions)
+	firstExecutionTime := closedExecutions[0].GetExecutionTime()
+	for i := 1; i != 4; i++ {
+		executionInfo := closedExecutions[i]
+		executionTime := executionInfo.GetExecutionTime()
+		s.Equal(int(0), int(executionTime/1000000000 - firstExecutionTime/1000000000 ) % 3)
+
+	}
 	for i := 0; i != 4; i++ {
 		executionInfo := closedExecutions[i]
 		s.Equal(targetBackoffDuration.Nanoseconds(), executionInfo.GetExecutionTime()-executionInfo.GetStartTime())
```

Comment:
Do we still need this check?

Reply:
and the check on L1021?

Reply:
Removed them

Reply:
Sorry, I think I was wrong here. The check on L1021 is testing whether the execution time in the result of DescribeWorkflow is correct or not. Since it's describing the first run, the difference between start and execution time is always 3s.

Reply:
Correct. L1010is duplicated. We still need L1021.